Doc/glossary.rst

```diff
@@ -203,6 +203,16 @@ Glossary
       A list of bytecode instructions can be found in the documentation for
       :ref:`the dis module <bytecodes>`.
 
+   callable
+      A callable is an object that is possible to execute, possibly with a set
```

Comment:
```suggestion
      A callable is an object that can be called, possibly with a set
```
A little circular but the additional term "execute" only makes it more confusing.